moto/ec2/resources/instance_type_offerings/region/ap-southeast-3.json

```diff
@@ -0,0 +1 @@
.2xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "m5d.8xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "r4.2xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "i3en.12xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "g4dn.2xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "m1.large", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "a1.large", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "r5b.metal", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "i2.4xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "u-12tb1.112xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "r6g.medium", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "c4.2xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "r5.large", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "m5a.24xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "g5g.xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "i3en.large", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "t3a.micro", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "c6g.8xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "r5ad.xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "c3.2xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "x1e.8xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "i3en.2xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "m5a.12xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "r5ad.8xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "m5zn.12xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "m5zn.3xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "c3.8xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "d2.8xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "i3.16xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "m6gd.12xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "r5dn.24xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "m5n.16xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "g3.4xlarge", "Location": "ap-southeast-3", "LocationType": "region"}, {"InstanceType": "m3.xlarge", "Location": "ap-southeast-3", "LocationType": "region"}]
```

Comment:
I'm thinking this may have been what was missing to make those Terraform tests happy?

All I did was make a copy of `ap-southeast-1.json` and did  find and replace from `southeast-1` to `southeast-3`, is there anything else in here that would need changed?